interface-definitions/include/firewall/common-rule-bridge.xml.i

```diff
@@ -0,0 +1,39 @@
+<!-- include start from firewall/common-rule-bridge.xml.i -->
+#include <include/firewall/action-l2.xml.i>
+#include <include/firewall/nft-queue.xml.i>
+<node name="destination">
+  <properties>
+    <help>Destination parameters</help>
+  </properties>
+  <children>
+    #include <include/firewall/mac-address.xml.i>
+  </children>
+</node>
+<leafNode name="disable">
+  <properties>
+    <help>Option to disable firewall rule</help>
```

Comment:
I would say just "Disable the rule", less wordy.

Reply:
Why not use: `#include <include/generic-disable-node.xml.i>`